common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -5356,6 +5358,14 @@ object CelebornConf extends Logging {
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("512k")
 
+  val CLIENT_SPARK_SHUFFLE_INTEGRITY_CHECK_ENABLED: ConfigEntry[Boolean] =
```

Comment:
Maybe better to revert back to using `celeborn.client.shuffle.integrityCheck.enabled` , as the configuration use in many place not only impact spark, As a compromise, just comment this configuration is only affects spark

Reply:
Ack. Updating